src/mlpack/methods/perceptron/perceptron_main.cpp

```diff
@@ -126,8 +133,14 @@ PARAM_MODEL_OUT(PerceptronModel, "output_model", "Output for trained perceptron"
 
 // Testing/classification parameters.
 PARAM_MATRIX_IN("test", "A matrix containing the test set.", "T");
+/* 
+* The PARAM_UROW_OUT("output") is depracated and
+* can be removed in mlpack4.0.0.
```

Comment:
Same comments here about the same comment in the previous file.

Reply:
Wrapped up line